src/dns/aardvark.rs

```diff
@@ -327,6 +327,61 @@ impl Aardvark {
         Ok(())
     }
 
+    // Modifies network dns_servers for a specific network and notifies aardvark-dns server
+    // with the change.
+    pub fn modify_network_dns_servers(
+        &mut self,
+        network_name: &str,
+        network_dns_servers: &Vec<String>,
+    ) -> Result<()> {
+        let mut dns_servers_modified = false;
+        let path = Path::new(&self.config).join(network_name);
+        let file_content = fs::read_to_string(&path)?;
+
+        let mut file = File::create(&path)?;
+
+        //for line in lines {
+        for (idx, line) in file_content.split_terminator('\n').enumerate() {
+            let mut content = line.to_string();
```

Comment:
this will cause an extra copy for each single line, no?
how about
```
if idx > 0 { write_all(line.as_bytes()); return}
```
and then handle the other case below

Reply:
will not copy now.